src/YQPkgFilterTab.cc

```diff
@@ -40,8 +40,9 @@
 #define USE_MYRLYN_LOGO  1
 
 
-struct YQPkgFilterTabPrivate
+class YQPkgFilterTabPrivate
```

Comment:
This is actually the same thing. A C++ `struct` is a `class` where everything is `public`. This was intentional. Why do we need to change it?

Reply:
To match the declaration in `src/YQPkgFilterTab.h` where it is class. I do not mind `class` or `struct` but it should be consistent.

Reply:
A forward declaration can use `class` and `struct` as synonyms. It doesn't and shouldn't reveal too many implementation details; just enough to tell the compiler that this is a large thing that I use pointers to in the header file.

So no, this is not an inconsistency, it's just not giving away implementation details. "Need to know" etc.

Reply:
I agree, there is no need to change. There is some weird combination on Windows resulting in linker errors, it should not matter for Myrlyn.

There is no need to not apply the change. It is the only case of diverging declaration and it clutters the Clang output -- this makes it harder to spot real problems. Can I persuade you to apply the change?

Reply:
_sigh..._

OK then.